### PR TITLE
#23 - Add basic for uploading zip archive

### PR DESCRIPTION
**Pull request by genryxy**
Part of #23 
Started to implement uploading zip archive

### CODE REVIEW

File: src/main/java/com/artipie/composer/http/AddZipSlice.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+/**
+ * Slice for adding a package to the repository in ZIP format.
+ * @since 0.4
+ */
+@SuppressWarnings({"PMD.SingularField", "PMD.UnusedPrivateField"})
+final class AddZipSlice implements Slice {
+    /**
+     * Composer HTTP for entry point.
+     */
+    public static final Pattern PATH = Pattern.compile("^/(?<name>.*)-(?<version>.*).zip$");
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/composer/http/AddZipSlice.java:48]: @genryxy what are the rules for names and versions in composer? Version, for example, can [contain](https://en.wikipedia.org/wiki/Software_versioning) `-`: 
/project-1.2.0-b.2.zip
in this case, name = project-1.2.0, version = b.2

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy have a concern about regular expression, please, take a look

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks

**Comment by 0crat** [comment]
Job `gh:artipie/composer-adapter#88` is not assigned, can't get performer

**Pull request merged**